### ISSUE TITLE
Xcode Build fail

### ISSUE THREAD

**Issue by Little0ne95** [ttatarinov/lineapro-phonegap-plugin-example#1]
After downloading this sample project, and running it from xcode, i receive around 13 errors. shown in the image.

any help to fix would be greate

![example](https://cloud.githubusercontent.com/assets/12908156/8854127/47374ac8-3156-11e5-8029-136a34fd920c.jpg)

**Comment by sclearion** [comment]
Same here. This plugin just doesn't work. Tried on 3 diff devices

**Comment by Little0ne95** [comment]
I have a fully working code for the Lina pro, and a guide on how to use it. I created it when I finally got mine working

**Comment by Little0ne95** [comment]
Hi I have this working now, if you want help?

**Comment by sclearion** [comment]
yes please. I followed the instructions verbatim and have the code in the ionic platform ready like:

``` js
$ionicPlatform.ready(function () {
        // hide the accessory bar by default on mobile deployments
        // (remove this to show the accessory bar above the keyboard for form inputs)
        if (window.cordova && window.cordova.plugins.Keyboard) {
            cordova.plugins.Keyboard.hideKeyboardAccessoryBar(true);
            cordova.plugins.Keyboard.disableScroll(true);
        }
        if (window.StatusBar) {
            StatusBar.styleDefault();
        }

        if (typeof LineaProCDV !== 'undefined') {
            try {
                LineaProCDV.initDT();
                alert('LineaProCDV Active');
            } catch (e) {
                alert('Error running LineaProCDV.initDT()');
            }
        } else {
            alert('LineaProCDV is not defined. You must not be on a mobile platform');
        }
    });
```

it does not even fire so I wonder what else I can do

**Comment by sclearion** [comment]
Can you send me the guide please :) :)

**Comment by Little0ne95** [comment]
https://github.com/Little0ne95/Linea-Pro-Rachel-v1

If you want you can contact me through my email, I'll be happy to help with any problem. If you do want my email let me know :)

**Comment by sclearion** [comment]
omg you rock. Thanks champ!

**Comment by Little0ne95** [comment]
If you don't understand the code, or my guide i have a zip file here with the code all ready to be used

----- Reply message -----

**Comment by sclearion** [comment]
No it was really self explanatory. Im just very new to apple development
being an android buy

**Comment by Little0ne95** [comment]
I had never done any app development at all, I just know a lot of java and Web development, got thrown in the deep end, but was a great learning curve. and since I couldn't find any help online when I did succeed I decided to put it together and put it online

----- Reply message -----

**Comment by sclearion** [comment]
You just  made me a hero here!!! I got it working!!! Thank you sooo much

**Comment by Little0ne95** [comment]
Glad I could help! Makes me feel good that I could help :)

----- Reply message -----
